themes/default/content/docs/intro/console/stack-permissions.md

```diff
@@ -1,51 +0,0 @@
----
-title: Stack Permissions
-meta_desc: An overview of Stack Permissions within the Pulumi Cloud Service.
-menu:
-  intro:
-    parent: console
-    weight: 5
-aliases:
-- /docs/reference/service/roles-and-access-controls/
-- /docs/console/collaboration/stack-permissions/
----
-
-The Pulumi Console provides fine-grained access controls for stacks.
```

Comment:
This is a good introductory line that it'd be nice to keep if we could. Right under the `Stack Permissions` heading would probably be a good place for it.